website/docs/reference/node-selection/configure-state.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Configure state selection"
+description: "Learn how to configure state selection in dbt."
+pagination_next: "reference/node-selection/state-comparison-caveats"
+---
+
+State and defer can be set by environment variables as well as CLI flags:
+
+- `--state` or `DBT_STATE`: file path
+- `--defer` or `DBT_DEFER`: boolean
```

Comment:
As mentioned above, link to the larger "defer" doc here and remove other references in this doc so they don't get confused